python/strings_udf/cpp/src/strings/udf/shim.cu

```diff
@@ -227,3 +228,45 @@ extern "C" __device__ int udf_string_from_string_view(int* nb_retbal,
 
   return 0;
 }
+
+extern "C" __device__ int strip(int* nb_retval,
```

Comment:
In the interest of expediency I'm fine merging this as is, but in the long run I suggest that we use some templating or macros to reduce the boilerplate in this file. There are a lot of functions with nearly identical signatures.

Reply:
To be clear, I recognize that the functions that numba is going to call need to use the C ABI, but we can still generate them with a macro or have them internally call a templated function. Not necessary to do in this PR though.

Reply:
Going to resolve this conversation for now but experiment with this a bit for next release 👍